### ISSUE TITLE
Feature: Show all "Get Access" property options

### ISSUE THREAD

**Issue by thansidwell** [CodeForPhilly/vacant-lots-proj#507]
Right now when you look at a property we display a Get Access option. 

![Image](https://github.com/CodeForPhilly/vacant-lots-proj/assets/1965986/18ec126d-1e7a-4c05-8b68-a6a65eeae915)

This would likely be interprited as the **_only_** option avaliable, but really there are other options avaliable this is just what we deem the ideal option. 

Make these changes to the section matching the [prototype](https://www.figma.com/file/NAFkgq34abW6uJ0R7PW24T/Prototype---Clean-%26-Green-Philly?type=design&node-id=2592%3A30687&mode=design&t=TMx1kZ7ygrhbLR1X-1).

- Reword description text
- Display the 6 panels as in the prototype 
  - icon, heading, description, badge and right arrow icon (phosphor icons) 
- Label each panel with colored badges based on the data
  - Best Option, Available, Not Available
- The "Purchase as Side Yard" is a special case. It should basically be static. It's always applicable, but just for neighborhoods. 
- Sort the panels in this order. Best Option, Avaliable, For Neighbors, and Unavaliable. 
- Color code the panels to match the badges
- When you click on one of these send users to the get Access Page and ideally scroll them down the associated section

![Image](https://github.com/CodeForPhilly/vacant-lots-proj/assets/1965986/00920628-7811-4be2-95ce-b36d21d47589)

**Comment by AZBL** [comment]
@thansidwell I'd like to be assigned this please.

**Comment by nlebovits** [comment]
Hey @AZBL, currently the data are stored in a few different columns. You may have to do some basic data manipulation in JS rather than work directly with the raw data as in the filters. Conservatorship, side yard eligibility, and quick cleaning are all individual columns (the latter corresponds to `tactical_urbanism`), while the other three options are all found in `access_process`. You might consider something like concatenating them as a list of strings before passing that to the component.

**Comment by AZBL** [comment]
Hey @nlebovits I think I need some clarification on this ticket request. For this request are the 6 cards supposed to be dynamically rendered for each property based on data we have, or should the cards look identical to the above for each property?

**Comment by thansidwell** [comment]
@AZBL Each property should have 6 cards, BUT the badge, color, and order of them should change depending on whether that access method is available, unavailable, or the best option.

**Comment by paulhchoi** [comment]
@moylena, I can take this one, please feel free to shift to "In Dev"

**Comment by paulhchoi** [comment]
@moylena, I can take this one, please feel free to shift to "In Dev"

**Comment by paulhchoi** [comment]
hey @thansidwell / @nlebovits, I'm double checking some of the logic and could use some clarifications:

1. Should the "best" option should be whatever the `access_process` is? (either "Private Land Use Agreement", "Land Bank", or "Buy Property")

2. There are a few properties where the "access_process" is "Do Nothing". How should we handle these properties?

3. if `access_process` is not "Private Land Use Agreement", should we display "Private Land Use Agreement" as "Unavailable?". Are there any cases where "Private Land Use Agreement" should be available, even if its not stated as the `access_process`?

4. if `access_process` is not "Land Bank", should we display "Land Bank" as "Unavailable?". Are there any cases where "Land Bank" should be available, even if its not stated as the `access_process`?

5. The sideyard is a bit confusing — the acceptance criteria said to just leave it static yellow for neighbors, but there is a filter option for "side_yard_eligible : No" — shouldn't we display the sideyard as "Unavailable" for those properties?

6. What should the "Quick Cleaning" link to on the `/get-access` page? Seems like everything has a logical match except for that one

**Comment by thansidwell** [comment]
@paulhchoi @nlebovits 

I think the `access process` status is calculated by looking other data, and was meant to be the only thing we would display. Now we realize it's more useful and logical to show all the options and whether you can do them or not. I wonder if we need to rework or reinterpret the data in the database. I worry they are trying to infer things from a single calculation `access process` when we really should just have a data point for each of these get access options as well as a `best option`. 

1. Yes, the "best option" is what's in the access process.
2. Perhaps when the access process is "do nothing" the other options will be "unavailable" so that will just imply or clarify that you can't do anything. 
3. I don't think you can infer if it's available or unavailable from the `get access` data point. It sounds like we need to rework the data, and perhaps have a simple `private land use agreement` column in the database. 
4. Same here. 
5. Oh! I forgot about that. If it is truly unavailable then we should give it the "unavailable" status. 
6. As I said in the pull request, we haven't written the panel for "Quick Cleaning/Tactical Urbanism." @nlebovits is primarily in charge of it.

**Comment by paulhchoi** [comment]
@thansidwell Thanks for the detailed response! so for 2, say for property #252158600: https://vacant-lots-proj-git-paulhchoi-50-4c3690-clean-and-green-philly.vercel.app/find-properties/252158600, the data returned is: 

```
{
    ...
    "OPA_ID": 252158600,
    ...
    "side_yard_eligible": "Yes",
    "tactical_urbanism": "Yes",
    ...
    "access_process": "Do Nothing"
}
```

so even though sideyard and quick cleaning is eligible, I should take `access_process` of `Do Nothing` as the most accurate, list Do Nothing as the best option, and overwrite everything else to be "Unavailable"?

**Comment by thansidwell** [comment]
@nlebovits @brandonfcohen1 @paulhchoi 

To complete how we are displaying all the get access options, Paul has to make a lot of assumptions on the data in the database. See he comments above. 

I wonder if we should just rework the database to better match the data shown on the front-end. Something like this perhaps? 

```
private land use agreement: yes/no
lank bank: yes/no
buy affordably from owner: yes/no
side yard program: yes/no
tactical urbanism: yes/no
conservatorship: yes/no
```

**Comment by thansidwell** [comment]
@paulhchoi  To your question, I hadn't thought of that! If it is sideyard and quick cleaning elligable we should definately be displaying that. Perhaps `access process` could change to `best access process` and `Do Nothing` could change to `None`, because, in reality, there are options, there's just no ideal option.

**Comment by brandonfcohen1** [comment]
> @nlebovits @brandonfcohen1 @paulhchoi
> 
> We want to now show all the get access options.
> 
> <img alt="CleanShot 2024-04-18 at 23 15 23@2x" width="891" src="https://private-user-images.githubusercontent.com/1965986/323820722-28a68985-88fa-48bc-ba8d-8b8842c64864.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM1MzAyMTEsIm5iZiI6MTcxMzUyOTkxMSwicGF0aCI6Ii8xOTY1OTg2LzMyMzgyMDcyMi0yOGE2ODk4NS04OGZhLTQ4YmMtYmE4ZC04Yjg4NDJjNjQ4NjQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MTlUMTIzMTUxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjQ4OGI1Mzk0MTllZmQwMWVlMDc2OTViYzRkZmFmMGFjZjU0ZDRlZjZkZTlkMTNkMDMxNjJkNDhmNDQ0ZDQ2OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.YFKnZmHP6NrWfRK-ZE_m_LbyClsrSo9CxJWLOes6NjQ">
> But many of the options we don't have individual data points in the database. For some of them we only have `access process`. Which is really the `best access process`. Perhaps we change the name?
> 
> So Paul is wondering if and how he should make assumptions on many of them based on that one data point?
> 
> I wonder if we should just rework the database to better match the data shown on the front-end. Something like this perhaps?
> 
> ```
> private land use agreement: yes
> land bank: no
> buy affordably from owner: no
> side yard program: yes
> tactical urbanism: yes
> conservatorship: no
> 
> best access process: private land use agreement
> ```

will let @nlebovits answer this

**Comment by nlebovits** [comment]
Yes, we can/should rewrite the pipeline to make the database format more consistent with our new needs. @brandonfcohen1 and I can work on that this summer, along with several other changes I want to make.

The idea of the data structure was to keep mutually exclusive data in the same column, e.g., if a property is eligible for purchase from a private holder, it is by definition not available through the land bank. There is some ambiguity around this in terms of purchase vs. private land use agreement, which *technically* aren't mutually exclusive. But bear in mind that this tool is primarily aimed at community groups who likely will find purchases of over $1,000 financially impossible, even if they're technically feasible.

I'll pull my comment from above, which could be off base, but is how I would approach this:

> Currently the data are stored in a few different columns. You may have to do some basic data manipulation in JS rather than work directly with the raw data as in the filters. Conservatorship, side yard eligibility, and quick cleaning are all individual columns (the latter corresponds to tactical_urbanism), while the other three options are all found in access_process. You might consider something like concatenating them as a list of strings before passing that to the component.

Also, @paulhchoi please change "Quick Cleaning" to "Tactical Urbanism". @thansidwell and I have talked about this and I think the correct step here is to use the technical term so that people who aren't familiar with the concept and the risks entailed don't get the impression that they can just go trespassing left and right.

**Comment by nlebovits** [comment]
closed by #538 and #551
